api/v3/Contribution.php

```diff
@@ -626,13 +616,14 @@ function civicrm_api3_contribution_repeattransaction($params) {
       'financial_type_id',
       'contribution_status_id',
       'membership_id',
+      'payment_processor_id',
     ];
     $input = array_intersect_key($params, array_fill_keys($passThroughParams, NULL));
 
-    return _ipn_process_transaction($params, $contribution, $input, $ids);
+    return _ipn_process_transaction($params, $contribution, $input);
   }
   catch (Exception $e) {
-    throw new API_Exception('failed to load related objects' . $e->getMessage() . "\n" . $e->getTraceAsString());
```

Comment:
If an exception is thrown in `_ipn_process_transaction` the message ends up as `failed to load related objectsfailed to load related objects`...

A follow-up PR will remove this try/catch entirely because it's silly - ie. if an exception is thrown catch it and throw it again!